client/app/shared/action-button-group/action-button-group.component.js

```diff
@@ -26,6 +27,7 @@ function ComponentController (sprintf) {
   vm.$onInit = function () {
     angular.extend(vm, {
       isDisabled: angular.isUndefined(vm.isDisabled) ? false : vm.isDisabled,
+      onSaveLabel: angular.isUndefined(vm.onSaveLabel) ? __('Save') : __(vm.onSaveLabel),
```

Comment:
> Don’t use dynamic content inside __()
http://manageiq.org/docs/guides/i18n

`s = __(sprintf("My name is %s.", me.name));`
The above should correctly be:
`s = sprintf(__("My name is %s."), me.name);`